pkg/scheduler/framework/plugins/volumebinding/volume_binding.go

```diff
@@ -123,6 +126,27 @@ func (pl *VolumeBinding) EventsToRegister() []framework.ClusterEventWithHint {
 	return events
 }
 
+func (pl *VolumeBinding) isSchedulableAfterCSINodeChange(logger klog.Logger, pod *v1.Pod, oldObj, newObj interface{}) (framework.QueueingHint, error) {
+	oldCSINode, modifiedCSINode, err := util.As[*storagev1.CSINode](oldObj, newObj)
```

Comment:
The current implementation returns Skip when a new CSINode is created and it doesn't have migrated plugins annotation.
But, actually I suppose in such case, we should return Queue.

Can you double-confirm the implementation of the plugin and change like this if you agree with me?

```suggestion
	if newObj == nil {
		logger.V(5).Info("CSINode creation could make the pod schedulable")
		return framework.Queue, nil
	}
	oldCSINode, modifiedCSINode, err := util.As[*storagev1.CSINode](oldObj, newObj)
```

Reply:
You mean I didn't consider [this process](https://github.com/kubernetes/kubernetes/blob/9fc0315ce89d425dc838338018b0c9f4da7559d9/pkg/scheduler/framework/plugins/volumebinding/binder.go#L1109) enough, right?
I believe you are right.

Reply:
[511acd3](https://github.com/kubernetes/kubernetes/pull/125097/commits/511acd3bd63737721a31998441a72c6e4ed29eff)